libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -29,265 +27,279 @@
 
 #define POSCONTROL_ACCEL_Z                      250.0f  // default vertical acceleration in cm/s/s.
 
-#define POSCONTROL_LEASH_LENGTH_MIN             100.0f  // minimum leash lengths in cm
-
-#define POSCONTROL_DT_50HZ                      0.02f   // time difference in seconds for 50hz update rate
-#define POSCONTROL_DT_400HZ                     0.0025f // time difference in seconds for 400hz update rate
-
-#define POSCONTROL_ACTIVE_TIMEOUT_US            200000  // position controller is considered active if it has been called within the past 0.2 seconds
-
 #define POSCONTROL_VEL_ERROR_CUTOFF_FREQ        4.0f    // low-pass filter on velocity error (unit: hz)
 #define POSCONTROL_THROTTLE_CUTOFF_FREQ         2.0f    // low-pass filter on accel error (unit: hz)
 #define POSCONTROL_ACCEL_FILTER_HZ              2.0f    // low-pass filter on acceleration (unit: hz)
-#define POSCONTROL_JERK_RATIO                   1.0f    // Defines the time it takes to reach the requested acceleration
+
+#define POSCONTROL_DEFAULT_SHAPER_TC           0.25f    // default time constant of the kinimatic path generation in seconds
 
 #define POSCONTROL_OVERSPEED_GAIN_Z             2.0f    // gain controlling rate at which z-axis speed is brought back within SPEED_UP and SPEED_DOWN range
 
+#define POSCONTROL_RELAX_TC                     0.16f
```

Comment:
I guess all the other definitions get comments so this one should too

Reply:
Yes, it felt left out.